src/client/ui/styles.less

```diff
@@ -659,6 +670,55 @@
                 }
             }
         }
+
+        .selector-panel-toggle-button {
+            background: white;
+            position: absolute;
+            height: 21px;
+            width: 70px;
+            padding: 3px 5px;
```

Comment:
I think we can move paddings from button to span pseudo element to make Copy button and Hide Selector buttons look the same.
You only need to copy paddings from Copy button to your new button